platform_main_feature.info

```diff
@@ -16,13 +16,15 @@ dependencies[] = islandora_compound_object
 dependencies[] = islandora_solr
 dependencies[] = menu
 dependencies[] = node_export_features
+dependencies[] = node
 dependencies[] = node_export_features
```

Comment:
Looks like it was here previously, but `node_export_features` being here twice is... strange?

... Looks like it's been here since the initial commit, actually... Let's ignore for now, I guess.